packages/framework/aqueduct/src/data-object-factories/pureDataObjectFactory.ts

```diff
@@ -300,6 +300,9 @@ export class PureDataObjectFactory<
 	 * @param initialState - The initial state to provide to the created component.
 	 * @returns an object created by this factory. Data store and objects created are not attached to container.
 	 * They get attached only when a handle to one of them is attached to already attached objects.
+	 *
+	 * @deprecated - the issue with createRootInstance originally and now is that it does not allow the customer to
```

Comment:
#nit

> originally and now 

is redundant. Just say what the issue is.

Reply:
Updated